.github/workflows/dart.yml

```
@@ -58,13 +58,20 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-11]
+        os: [macos-12]
```

```suggestion
        os: [macos-11, macos-12]
```

Right? As the excludes are already in place, but the OS is not selected.

---

Added.